tests/containers/podman_netavark.pm

```diff
@@ -78,7 +80,7 @@ sub run {
         return 1;
     }
 
-    switch_to_netavark unless (is_alp || is_sle_micro('6.0+'));
+    switch_to_netavark if is_cni_default;
```

Comment:
That will not work because the previous module, `podman_network_cni` should have switched the network to CNI. See e.g. http://kepler.suse.cz/tests/22380.

In short: Currently `podman_network_cni` doesn't switch the network to CNI which is wrong. But when it does, then this one is broken, because it wouldn't switch back.